app/components/SideMenu.js

```diff
@@ -103,7 +103,7 @@ const styles = StyleSheet.create({
   },
   menu: {
     flex: 1,
-    width: DeviceScreen.width - MENU_OFFSET,
+    width: DeviceScreen.width - MENU_OFFSET + 20,
```

Comment:
I think we could add this to the MENU_OFFSET constant. 

@BrettBukowski is that ok?